versions.lock

```diff
@@ -5,37 +5,45 @@ com.fasterxml.jackson.core:jackson-databind:2.11.3 (3 constraints: 4a3a2999)
 com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.11.3 (1 constraints: cf0c0316)
 com.fasterxml.jackson.module:jackson-module-afterburner:2.11.3 (1 constraints: cf0c0316)
 com.github.ben-manes.caffeine:caffeine:2.9.0 (1 constraints: 0d051036)
+com.google.auto:auto-common:1.0 (2 constraints: 8b1626fd)
```

Comment:
Do we expect any downstream issues with this dep?

Reply:
We've used it several places including the dialogue annotation processor, I don't expect issues, but worst case we can hand-write the META-INF/services file

Reply:
Added a comment in build.gradle -- this dependency is used to add the correct `@Generated` annotation based on the target java version.